working-groups/rls-2.0/NOTES.md

```diff
@@ -57,3 +57,19 @@ While it is unrealistic to extract the full type-inference engine from the compi
 ## Summary
 
 In short, we decided to primarily work towards an end-to-end effort, which aims to _correctly_ support a _subset_ of the language features in the short term. However, as time goes, we plan on achieving feature parity with the compiler in the long run and merge the resulting code upstream.
+
+# 2019-04-11
+
+## Notes from "Macro By Example" meeting
+
+[Zulip stream](https://rust-lang.zulipchat.com/#narrow/stream/185405-t-compiler.2Fwg-rls-2.2E0/topic/MBE.20discussion)
+
+We've discussed current state and next steps for macro handling in rust-analyzer. One conclusion was that we are not going to shoot directly at librarifing macro expansion code and sharing it with rustc: we already have chalk and name resolution in-flight. Instead, the focus is explicitly on supporting the most common cases, so that we can check hypothesis that incremental IDE-style macro expansion is feasible.
```

Comment:
Small typo
```suggestion
We've discussed current state and next steps for macro handling in rust-analyzer. One conclusion was that we are not going to shoot directly at librarifying macro expansion code and sharing it with rustc: we already have chalk and name resolution in-flight. Instead, the focus is explicitly on supporting the most common cases, so that we can check hypothesis that incremental IDE-style macro expansion is feasible.
```